src/main/java/seedu/duke/data/timetable/Event.java

```diff
@@ -68,19 +73,27 @@ public void setTime(LocalTime time) {
         this.time = time;
     }
 
-    public Boolean getToRemind() {
+    public boolean getToRemind() {
         return isToRemind;
     }
 
-    public void setToRemind(Boolean toRemind) {
+    public void setToRemind(boolean toRemind) {
         isToRemind = toRemind;
     }
 
-    public Boolean getRecurring() {
+    public boolean getRecurring() {
         return isRecurring;
     }
 
-    public void setRecurring(Boolean recurring) {
+    public void setRecurring(boolean recurring) {
         isRecurring = recurring;
     }
+
+    public String toString() {
+        String titleString = "Event: " + title;
+        String dateString = "Date: " + date.toString() + "\tTime: " + time.toString();
+        String remindString = "Reminder: " + isToRemind;
+        String repeatingString = "Repeating: " + isRecurring;
+        return titleString + "\n" + dateString + "\n" + remindString + "\n" + repeatingString;
```

Comment:
Use InterfaceManager.LS to standardize for new lines.
```suggestion
        return titleString + InterfaceManager.LS 
            + dateString + InterfaceManager.LS 
            + remindString + InterfaceManager.LS
            + repeatingString;
```

Reply:
Done. Thanks!